sdk/identity/azure-identity/azure/identity/_internal/__init__.py

```diff
@@ -66,21 +64,20 @@ def validate_tenant_id(tenant_id):
         )
 
 
-def resolve_tenant(default_tenant, allow_multitenant, tenant_id=None, **_):
-    # type: (str, bool, Optional[str], **Any) -> str
+def resolve_tenant(default_tenant, tenant_id=None, **_):
```

Comment:
This could be simpler:
 ```py
if (
    tenant_id is None
    or default_tenant == "adfs"
    or os.environ.get(EnvironmentVariables.AZURE_IDENTITY_DISABLE_MULTITENANTAUTH)
):
    return default_tenant
return tenant_id
```